smac/runhistory/runhistory.py

```diff
@@ -314,7 +323,7 @@ def update_from_json(self, fn, cs):
         """
         new_runhistory = RunHistory(self.aggregate_func)
         new_runhistory.load_json(fn, cs)
-        self.update(runhistory=new_runhistory)
+        self.update(runhistory=new_runhistory, external_data=True)
```

Comment:
I think it should be an argument of the function whether the new data is treated as external_data